calculator.go

```diff
@@ -1,13 +1,112 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"fmt"
+	"math"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// Add takes 2+ numbers and returns the result of adding them together.
+func Add(a, b float64, rest ...float64) float64 {
+	result := a + b
+	for _, item := range rest {
+		result += item
+	}
+
```

Comment:
BIT house style is to write no blank lines within functions. (Use as many blank lines as you want in non-BIT projects; this is really to just help students achieve a consistent and Go-like style before they start to develop their own.)